README.md

```diff
@@ -36,9 +39,9 @@ You can configure it by overriding the `scale.minAvailable` parameter. (e.g. `90
 
 ## Tests
 
-There are multiple complementary ways to test the chart before relesing it. We have defined workflow that uses the `chart-testing` (`ct`) library to:
+On PR, a github workflow is run which validates and deploys the app-helm-chart on a k8s cluster using
+various value files (take files ending with `-values.yaml` in the [ci directory](ci)). Workflow is copied from <https://github.com/renovatebot/helm-charts>.
```

Comment:
it takes any .yaml file 
https://github.com/DND-IT/app-helm-chart/pull/47/files#diff-43bbc03c76533ab7bc7c06bfda6378f2622e1b2ad10ff9244eb90f3f495fb97bR34

Reply:
ah good to know. thanks for the update